csaf_2.1/prose/edit/src/tests-01-mndtr-31-version-range-in-product-version.md

```diff
@@ -3,22 +3,28 @@
 For each element of type `/$defs/branches_t` with `category` of `product_version` it MUST be tested that
 the value of `name` does not contain a version range.
 
-> To implement this test it is deemed sufficient that, when converted to lower case,
-> the value of `name` does not contain any of the following strings:
+> To implement this test it is deemed sufficient that, when converted to lower case, the value of `name` satisfies the two requirements below:
 >
-> ```
->   <
->   <=
->   >
->   >=
->   after
->   all
->   before
->   earlier
->   later
->   prior
->   versions
-> ```
+> 1. It does not contain any of the following operators:
+>
+>    ```
+>      <
+>      <=
+>      >
+>      >=
+>    ```
+>
+> 2. If interpreted as individual word separated by whitespace, it does not contain any of the following keywords:
```

Comment:
```suggestion
> 2. If interpreted as a list of individual words separated by whitespace, the list does not contain any of the following keywords:
```

Reply:
I tried to rephrase it a little bit more.

Reply:
Works for me as well.